cmd/zetae2e/stress.go

```diff
@@ -337,21 +299,41 @@ func EchoNetworkMetrics(runner *runner.E2ERunner) {
 	}
 }
 
-func WithdrawETHZRC20(runner *runner.E2ERunner) {
+func WithdrawETHZRC20(r *runner.E2ERunner) {
 	defer func() {
 		zevmNonce.Add(zevmNonce, big.NewInt(1))
 	}()
 
-	ethZRC20 := runner.ETHZRC20
+	ethZRC20 := r.ETHZRC20
+	r.ZEVMAuth.Nonce = zevmNonce
 
-	runner.ZEVMAuth.Nonce = zevmNonce
-	_, err := ethZRC20.Withdraw(runner.ZEVMAuth, local.DeployerAddress.Bytes(), big.NewInt(100))
-	if err != nil {
-		panic(err)
-	}
+	must(ethZRC20.Withdraw(r.ZEVMAuth, local.DeployerAddress.Bytes(), big.NewInt(100)))
 }
 
 // Get ETH based chain ID
 func getChainID(client *ethclient.Client) (*big.Int, error) {
 	return client.ChainID(context.Background())
 }
+
+func must[T any](v T, err error) T {
```

Comment:
Should we just define these `must/exit/ensure` methods in `testutil` package?

Reply:
Let's keep this as is for now. If we have more duplicates, we can indeed move it and use smth like an alias:
```go
var must = testutil.Must
```